sdk/nodejs/tests/util.ts

```diff
@@ -14,12 +14,17 @@
 
 import assert from "assert";
 
-/** @internal */
-export type MochaFunc = (err: Error) => void;
+// NOTE: These helpers should no longer be used by tests.
+// This module remains as part of a closure serialization test that serializes this module.
```

Comment:
I'd wanted to delete this file altogether, but we use it in the "Capture module through indirect function references" closure serialization test:

https://github.com/pulumi/pulumi/blob/fdb5ea2951a718d4c821bd1ec149b68c7bc1ac70/sdk/nodejs/tests/runtime/tsClosureCases.ts#L944-L954

So I'll keep it around for that for now. But I could imagine converting that test to using a module that's actually a module just for testing.